CHANGELOG.md

```diff
@@ -858,6 +858,8 @@ Non-backwards compatible changes
 
 * In `Data.Sum.Base` the definitions `fromDec` and `toDec` have been moved to `Data.Sum`.
 
+* In `Data.Vec.Base`: the definitions `init` and `last` have been replaced by an inductive definition.
```

Comment:
Suggest replrasing:
"... replaced by a direct recursive definition in place of the `InitLast` view-derived definitions"

Reply:
Done. Thanks!